package.json

```diff
@@ -1,34 +1,39 @@
 {
-    "name": "dependo",
```

Comment:
after a `npm install` tis diff got big all of a sudden. I just added 2 keywords and one dependency